srcpkgs/wlrobs/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'wlrobs'
+pkgname=wlrobs
+version=v1.0
+revision=1
+build_style=meson
+hostmakedepends="cmake pkg-config"
+makedepends="wayland-devel obs-devel libX11-devel"
+short_desc="OBS plugin to screen capture wlroots based wayland compositors"
```

Comment:
What's the difference between this plugin and a proper setup with pipewire and portals?